components/TwItemView.js

```diff
@@ -40,7 +41,7 @@ class TwItemView extends React.Component {
         <Text note style={styles.title}>Description</Text>
         <Text>{alertItem.DESCRIPTION.replace(/<\/?[^>]+(>|$)/g, '')} </Text>
         <Text note style={styles.title}>Date</Text>
-        <Text>{new Date(alertItem.STARTDATE).toLocaleDateString()}</Text>
+        <Text>{formatDate(new Date(alertItem.STARTDATE))}</Text>
```

Comment:
Do we need a custom function for this? Would `new Date(alertItem.STARTDATE).toLocaleDateString('en-GB')` work?

Reply:
That could work, but I don't believe it adds the extra 0's which Kat desired. So the custom function works to ensure that the date is formatted in that exact configuration.

Reply:
Hi @gkemp94 , 
can you check what the call @lukekearney proposed would display? 
Just testing it locally and attaching a photo here is sufficient for me.

Reply:
@SmilingKat committing change now.